modules/src/Microsoft.Azure.IIoT.Modules.Discovery/src/Controllers/DiscoverySettingsController.cs

```diff
@@ -98,6 +100,14 @@ public class DiscoverySettingsController : ISettingsController {
             get => _discovery.Configuration.IdleTimeBetweenScans;
         }
 
+        /// <summary>
+        /// Discovery Urls to scan
+        /// </summary>
+        public Dictionary<string, string> DiscoveryUrls {
+            set => _discoveryUrls = value;
```

Comment:
Consider doing it like above - _config.DiscoveryUrls = value.DecodeAsList(), get=> _config.DiscoveryUrls.EncodeAsList().   Then remove the newly added member and logic below.

Reply:
changed.
I did that way because I used as a sample DiscoverySettingController.cs